src/Microsoft.AspNet.Cors.Core/CorsOptions.cs

```diff
@@ -13,8 +13,30 @@ namespace Microsoft.AspNet.Cors.Core
     /// </summary>
     public class CorsOptions
     {
+        private string _defaultPolicyName = "__DefaultCorsPolicy";
         private IDictionary<string, CorsPolicy> PolicyMap { get; } = new Dictionary<string, CorsPolicy>();
 
+        public string DefaultPolicyName
+        {
+            get
+            {
+                return _defaultPolicyName;
+            }
+
+            set
+            {
+                if (value == null)
+                {
+                    throw new ArgumentNullException("value",
+                                                    Resources.FormatPropertyOfTypeCannotBeNull(
```

Comment:
Yeah those comments were incorrect. We shouldn't be using any special messages for such common exceptions.